cmd/avalanche.go

```diff
@@ -31,6 +31,7 @@ var (
 	remoteRequestCount  = kingpin.Flag("remote-requests-count", "how many requests to send in total to the remote_write API.").Default("100").Int()
 	remoteReqsInterval  = kingpin.Flag("remote-write-interval", "delay between each remote write request.").Default("100ms").Duration()
 	remoteTenant        = kingpin.Flag("remote-tenant", "Tenant ID to include in remote_write send").Default("0").String()
+	remoteAuthHeader    = kingpin.Flag("remote-url-auth", "Authorization header to pass in for the remote-url").String()
```

Comment:
I would say just `remote-auth` for the flag, `url` does not add much. The help text should probably also specify that this is the value of the Authorization header.